src/Microsoft.AspNetCore.Mvc.Core/ApplicationParts/ViewInfoContainer.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.Mvc.ApplicationParts
+{
+    /// <summary>
+    /// A container for <see cref="ViewInfo"/> instances.
+    /// </summary>
+    public class ViewInfoContainer
```

Comment:
Just double checking, this name change hasn't been updated in the other PR yet right?